### DISCUSSION TITLE
Integrating Google Play Services in a rust android app?

### DISCUSSION THREAD

**Discussion by JasonMiesionczek** [rust-mobile/.github#1] [Q&A] [answered]
Hi there,

I was wondering if anyone has any information about how someone might go about using the google play services APIs from within a rust-based android app? Specifically from the point of view of a bevy game. I understand they are mostly just API calls, I know there are client libraries in java that can be included via the gradle build, but i'm not sure if those can/need to be referenced at all in order for things like google sign-in to work.

Appreciate any insight anyone can offer. 

Thanks.

**Comment by MarijnS95** [comment, marked as answer]
I recently ran into https://github.com/dodorare/crossbow which seems to provide a bunch of Google Play-related crates.

**Reply by JasonMiesionczek** [reply]
holy crap, that looks perfect, thanks!

**Comment by NiklasEi** [comment]
@JasonMiesionczek Did you manage an integration of google play services in your bevy app? If so, is there any open source code I could take a look at?
